src/main/resources/application.yml

```diff
@@ -8,7 +8,7 @@ spring:
     continue-on-error: true
     username: ${DB_USERNAME:postgres}
     password: ${DB_PASSWORD:postgres}
-    url: jdbc:postgresql://${DB_HOST:localhost}:${DB_PORT:5432}/${DB_DATABASE:db}
+    url: jdbc:postgresql://${DB_HOST:localhost}:${DB_PORT:5432}/${DB_DATABASE:postgres}
```

Comment:
Was it a necessary change?

Reply:
Sorry, this is my fault. Changed for local testing, forgot to change back